editor/editor_node.cpp

```diff
@@ -8045,6 +8048,7 @@ EditorNode::EditorNode() {
 	project_menu->add_item(TTRC("Pack Project as ZIP..."), PROJECT_PACK_AS_ZIP);
 #ifndef ANDROID_ENABLED
 	project_menu->add_item(TTRC("Install Android Build Template..."), PROJECT_INSTALL_ANDROID_SOURCE);
+	project_menu->add_item(TTR("Open Project Folder"), PROJECT_OPEN_PROJECT_FOLDER);
```

Comment:
As you pushed update anyway.
```suggestion
	project_menu->add_item(TTRC("Open Project Folder"), PROJECT_OPEN_PROJECT_FOLDER);
```